src/FontDescriptor.h

```diff
@@ -75,17 +76,17 @@ struct FontDescriptor {
   }
 
   ~FontDescriptor() {
-    if (path)
-      delete path;
+    //if (path)
```

Comment:
Since these are allocated by R we can't free them in the destructor. If you want to have FontDescriptor handle the memory you would have to allocate the memory on the stack yourself with malloc.

Reply:
will it be handled gracefully if I just remove the deletes? In the constructor they use `copyString` which uses `new` - shouldn't it always be coupled with a `delete`

Reply:
Maybe something else is going on then, I was getting double free errors and didn't look into it a great deal.